src/main/java/com/onarandombox/multiverseinventories/share/Sharables.java

```diff
@@ -582,6 +590,149 @@ public boolean updatePlayer(Player player, PlayerProfile profile) {
             }).serializer(new ProfileEntry(false, "potions"), new PotionEffectSerializer())
             .altName("potion").altName("potions").build();
 
+    /**
+     * Sharing Advancements.
+     */
+    public static final Sharable<List> ADVANCEMENTS = new Sharable.Builder<List>("advancements", List.class,
```

Comment:
Looks like you are casting into a Set of most of the updateProfile and updatePlayer code, maybe a Set datatype for this Sharable will be better?

Reply:
Same reason as above. If we set the datatype to Set, we'll need to write a serializer.